website/docs/Examples/Integrate - Spark.md

```diff
@@ -0,0 +1,112 @@
+# Integrate - Spark
+
+FLAML has integrated Spark for distributed training. There are two main aspects of integration with Spark:
+- Use Spark ML estimators for AutoML.
+- Use Spark to run training in parallel spark jobs.
+
+## Spark ML Estimators
+
+FLAML integrates estimators based on Spark ML models. These models are trained in parallel using Spark, so we called them Spark estimators. To use these models, you first need to organize your data in the required format.
+
+### Data
+
+For Spark estimators, AutoML only consumes Spark data. FLAML provides a convenient function `to_pandas_on_spark` in the `flaml.automl.spark.utils` module to convert your data into a pandas-on-spark (`pyspark.pandas`) dataframe/series, which Spark estimators require.
+
+This utility function takes data in the form of a `pandas.Dataframe` or `pyspark.sql.Dataframe` and converts it into a pandas-on-spark dataframe. It also takes `pandas.Series` or `pyspark.sql.Dataframe` and converts it into a [pandas-on-spark](https://spark.apache.org/docs/latest/api/python/user_guide/pandas_on_spark/index.html) series. If you pass in a `pyspark.pandas.Dataframe`, it will not make any changes, so don't worry about multiple calls with the same data.
```

Comment:
What does it mean by "multiple calls with the same data"?

Reply:
It means like you have already converted your data into pandas-on-spark format by using `to_pandas_on_spark` but once again you call `to_pandas_on_spark` on your data. It's a confusing sentence indeed, I might consider removing it.